python/apps/taiga/tests/unit/taiga/comments/test_services.py

```diff
@@ -14,35 +14,73 @@
 pytestmark = pytest.mark.django_db
 
 
+#####################################################
+# create_comment
+#####################################################
+
+
+async def test_create_comment():
+    project = f.build_project()
+    story = f.build_story()
+    comment = f.build_comment()
+
+    with (
+        patch("taiga.comments.services.comments_repositories", autospec=True) as fake_comments_repositories,
+        patch("taiga.comments.services.comments_serializers", autospec=True) as fake_comments_serializers,
+    ):
+        fake_comments_repositories.create_comment.return_value = comment
+
+        await services.create_comment(
+            project=project,
+            content_object=story,
+            text=comment.text,
+            created_by=comment.created_by,
+        )
+
+        fake_comments_repositories.create_comment.assert_awaited_once_with(
+            content_object=story,
+            text=comment.text,
+            created_by=comment.created_by,
+        )
+        fake_comments_serializers.serialize_comment.assert_called_once_with(comment=comment)
+
+
 #####################################################
 # list_comments
 #####################################################
 
 
 async def test_list_comments():
-    _story = await f.create_story()
-    _comment = await f.create_story_comment(content_object=_story)
+    story = f.build_story(id="")
+    comment = f.build_comment()
```

Comment:
Thanks for avoiding hitting the database. i'll take special care for the next unit tests.

Reply:
:+1: